aten/src/ATen/native/TensorTransformations.cpp

```diff
@@ -142,4 +142,45 @@ Tensor flipud(const Tensor& self) {
   return self.flip({0});
 }
 
+Tensor moveaxis(const Tensor& self, IntArrayRef src, IntArrayRef dst) {
+  TORCH_CHECK(src.size() == dst.size(), "Invalid Source or Destination Axes");
+  auto normalized_src = src.vec();
+  maybe_wrap_dims(normalized_src, self.dim());
+  auto normalized_dst = dst.vec();
+  maybe_wrap_dims(normalized_dst, self.dim());
+
+  auto it_src = std::unique(normalized_src.begin(), normalized_src.end());
+  TORCH_CHECK(it_src == normalized_src.end(), "repeated axis in `src` argument");
```

Comment:
You can actually print out `src` in the error message. Something like:
```
TORCH_CHECK(it_src == normalized_src.end(), "moveaxis: repeated axis in `src` (", src, ")");
```

Reply:
Done. Thanks!